src/com/itacademy/eshop/product/types/Stock.java

```diff
@@ -0,0 +1,7 @@
+package com.itacademy.eshop.product.types;
+
+public enum Stock {
+    InStock,
```

Comment:
Totally valid values, but enum naming convention is all capital letter separated by underscore for example IN_STOCK OUT_OF_STOCK